Room assignment task.md

```diff
@@ -0,0 +1,104 @@
+# Summary
+Participants had to assign students to rooms to maximize student utility.
+
+# References
+> Main paper: https://psyarxiv.com/ra9qy/
+
```

Comment:
The researchers have a really good overview of the experiment here: https://github.com/amaatouq/room-assignment/blob/master/Readme.md

Reply:
they also include all of their materials on this page: https://github.com/amaatouq/room-assignment

Reply:
Great find! What materials are you referring to?